projects/novo-elements/src/elements/form/extras/file/FileInput.ts

```diff
@@ -239,16 +306,30 @@ export class NovoFileInputElement implements ControlValueAccessor, OnInit, OnDes
     this.process(Array.from(event.target.files));
   }
 
+  validate(files): boolean {
+    let passedValidation = true;
+    if (this.layoutOptions.customValidation) {
+      this.layoutOptions.customValidation
```

Comment:
You could make this method a lot shorter by using `.every` here instead of `.forEach` since it would let you return a chain of array calls

Reply:
This is in Master already, different ticket